provisioning/install-config-visualizer.sh

```diff
@@ -1,18 +1,13 @@
 #!/usr/bin/env bash
 set -ex
 
-# Get the config-visualizer from GitHub
-if [ ! -d "config-visualizer/" ]; then
-    git clone --depth=1 --branch master https://github.com/precice/config-visualizer.git
-fi
-pip3 install --user -e config-visualizer
+# Install CLI dependencies
+sudo apt-get install -y graphviz
+# Install GUI depedencies
+sudo apt-get install -y libcairo2-dev libgirepository1.0-dev gcc libcairo2-dev pkg-config gir1.2-gtk-4.0
+
+# Get the config-visualizer from PIP
+pipx install precice-config-visualizer precice-config-visualizer-gui
```

Comment:
One package at a time.

```suggestion
pipx install precice-config-visualizer
pipx install precice-config-visualizer-gui
```